packages/spark/components/_masthead.scss

```diff
@@ -38,10 +38,11 @@
   transition: $sprk-masthead-transition;
 }
 
-// Keep padding settings consistent for menu, logo branding, and little nav
+// Keep padding settings consistent for menu, logo branding, little nav, and additional nav link
 .sprk-c-Masthead__menu,
 .sprk-c-Masthead__branding,
-.sprk-c-Masthead__little-nav {
+.sprk-c-Masthead__little-nav,
+.sprk-c-Masthead__nav-link {
```

Comment:
Open to different naming suggestions for this class. I just didn't want it to be sign on or sign off specific since it can actually be anything.

Reply:
It could be confusing since we have sprk-c-Masthead__site-links and sprk-c-Masthead__link already. Another thing that feels weird is that you are calling the link's container a link.

Reply:
Does sprk-c-Masthead__nav-item feel more appropriate?